### ISSUE TITLE
Video does not play inline on iOS[BUG]

### ISSUE THREAD

**Issue by Wian-TMC** [sarbagyastha/youtube_player_flutter#530]
**Describe the bug**
Using 2.2.1. The player does not play inline videos on iOS. It just shows a white block and does not play or load. No error warnings are visible in the logs. Tested using ios 14.5+

**To Reproduce**
Try play a video inline on iOS.

**Expected behavior**
The video should load and play inline. Works on Android

**Technical Details:**
 - Device: iPhone 12 Pro Max
 - OS: 14.5 simulator and 14.6 real device

**Images**
![Simulator Screen Shot - iPhone 12 Pro - 2021-08-06 at 10 33 19](https://user-images.githubusercontent.com/72802049/128482063-81f7948a-1742-477f-9417-fe792507093d.png)

Notice that the player has a white background, but is scrollable horizontally and vertically. But no video loads or plays.

**Additional context**
The system logs on the simulator when I try open the youTube player gives the following: Note - The last two lines might have some importance
```
Aug  6 10:27:23 Wians-MacBook-Pro syslogd[79405]: ASL Sender Statistics
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.CoreMLModelSecurityService): Unknown key for Boolean: EnableTransactions
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.CoreMLModelSecurityService): Unknown key for Boolean: EnablePressuredExit
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.DocumentManagerCore.Downloads): Unknown key for Boolean: EnableTransactions
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.DocumentManagerCore.Downloads): Unknown key for Boolean: EnablePressuredExit
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.FileProvider.ArchiveService): Unknown key for Boolean: EnableTransactions
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.FileProvider.ArchiveService): Unknown key for Boolean: EnablePressuredExit
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.FontServices.FontProviderLoader): Unknown key for Boolean: EnableTransactions
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.FontServices.FontProviderLoader): Unknown key for Boolean: EnablePressuredExit
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.FontServices.GSFontServices): Unknown key for Boolean: EnableTransactions
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.FontServices.GSFontServices): Unknown key for Boolean: EnablePressuredExit
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.FontServices.UserFontManager): Unknown key for Boolean: EnableTransactions
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.FontServices.UserFontManager): Unknown key for Boolean: EnablePressuredExit
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.MTLCompilerService): Unknown key for Boolean: EnableTransactions
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.MTLCompilerService): Unknown key for Boolean: EnablePressuredExit
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.StreamingUnzipService): Unknown key for Boolean: EnableTransactions
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.StreamingUnzipService): Unknown key for Boolean: EnablePressuredExit
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.WebKit.GPU): Unknown key for Boolean: EnableTransactions
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.WebKit.GPU): Unknown key for Boolean: EnablePressuredExit
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.WebKit.Networking): Unknown key for Boolean: EnableTransactions
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.WebKit.Networking): Unknown key for Boolean: EnablePressuredExit
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.WebKit.WebAuthn): Unknown key for Boolean: EnableTransactions
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.WebKit.WebAuthn): Unknown key for Boolean: EnablePressuredExit
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.WebKit.WebContent): Unknown key for Boolean: EnableTransactions
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.WebKit.WebContent): Unknown key for Boolean: EnablePressuredExit
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.backboard.display.archive): Unknown key for Boolean: EnableTransactions
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.backboard.display.archive): Unknown key for Boolean: EnablePressuredExit
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.intents.intents-helper): Unknown key for Boolean: EnableTransactions
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.intents.intents-helper): Unknown key for Boolean: EnablePressuredExit
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.textkit.nsattributedstringagent): Unknown key for Boolean: EnableTransactions
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.textkit.nsattributedstringagent): Unknown key for Boolean: EnablePressuredExit
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.uifoundation-bundle-helper): Unknown key for Boolean: EnableTransactions
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.uifoundation-bundle-helper): Unknown key for Boolean: EnablePressuredExit
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.xpc.launchd.domain.pid.WebContent.87617): Failed to bootstrap path: path = /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/System/Library/Frameworks/IOKit.framework/Versions/A/IOKit, error = 2: No such file or directory
Aug  6 10:27:23 Wians-MacBook-Pro com.apple.CoreSimulator.SimDevice.F4B9E2CE-76E0-4B62-A215-1FD1DB4FDA83[79403] (com.apple.xpc.launchd.domain.pid.WebContent.87617): Failed to bootstrap path: path = /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/usr/lib/dyld_sim, error = 2: No such file or directory
```

**Comment by PcolBP** [comment]
Give thumb up on #525, same problem.

**Comment by sarbagyastha** [comment]
Fixed with v2.2.2
